shared/redux/rtk-apis/enrollments/enrollments.api.ts

```diff
@@ -0,0 +1,13 @@
+import { projectApi } from "../api.config";
+import { IEnrollment } from "./enrollments.types";
+
+const enrollmentApi = projectApi.injectEndpoints({
```

Comment:
The enrollmentApi may be unnecessary. These are likely just classroom api endpoints